ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -7184,9 +7238,63 @@ components:
           required:
             - value
             - unit
-        grossVolume:
+        cargoGrossVolume:
+          type: object
+          title: Cargo Gross Volume
+          description: |
+            Calculated by multiplying the width, height, and length of the packed cargo.
+          properties:
+            value:
+              type: number
+              format: float
+              minimum: 0
+              exclusiveMinimum: true
+              description: |
+                Calculated by multiplying the width, height, and length of the packed cargo.
+              example: 2.4
+            unit:
+              type: string
+              description: |
+                The unit of measure which can be expressed in either imperial or metric terms
+                - `FTQ` (Cubic foot)
+                - `MTQ` (Cubic meter)
+              enum:
+                - MTQ
+                - FTQ
+              example: MTQ
+          required:
+            - value
+            - unit
+        cargoNetWeight:
+          type: object
+          title: Cargo Net Weight
+          description: |
+            The net weight of the cargo excluding packaging items being carried in the container(s). Excludes the tare weight of the container(s).
+          properties:
+            value:
+              type: number
+              format: float
+              minimum: 0
+              exclusiveMinimum: true
+              description: |
+                Net weight of the goods carried, excluding packaging. Excludes the tare weight of the container(s).
```

Comment:
Should the description be the same for the CargoNetWeight object and the value property?

Reply:
All descriptions now aligned with IFS